apps/vmq_server/priv/vmq_server.schema

```diff
@@ -1575,6 +1575,13 @@
                                                                             hidden
                                                                            ]}.
 
+%% @doc Sets the connection_rate_limit.
+{mapping, "max_conns_per_second", "vmq_server.max_conns_per_second", [
```

Comment:
Let's make it configurable on a per listener basis, similar to no. of acceptors, etc.